.github/ISSUE_TEMPLATE/workers-types.md

```diff
@@ -0,0 +1,10 @@
+---
+name: workers-types
+about: Report an issue or suggestion for workers-types
+title: ''
+labels: workers-types
+assignees: workers-devprod
```

Comment:
Do we want to assign it to the service account user? I suppose this will cause everyone on the team to get an email, which is pretty good

Reply:
Is it possible to assign to a team?

Reply:
@penalosa i don't see an option to assign to a team :/

Reply:
Yeah, as far as I know, github currently doesn't support assigning teams